README.md

```diff
@@ -7,46 +7,50 @@ database dump is provided that includes the following information:
 
 Information about ports, including:
 
-* 5-character Port Code
-* Port Name
-* Slug that describes which Region does the port belong in
+* 5-character port code
+* Port name
+* Slug describing which region the port belongs to
 
 ## Regions
 
-A hierarchy of Regions, including:
+A hierarchy of regions, including:
 
-* Slug - a machine-readable form of the Region name
-* The name of the Region
-* Slug that describes which parent Region does the Region belong in
+* Slug - a machine-readable form of the region name
+* The name of the region
+* Slug describing which parent region the region belongs to
 
 ## Prices
 
 Individual daily prices between ports, in USD.
 
-* 5-character Origin Port Code
-* 5-character Destination Port Code
-* The day on which the price is valid on
+* 5-character origin port code
+* 5-character destination port code
+* The day for which the price is valid
 * The price in USD
 
 # Assignment
 
 Develop an HTTP-based API capable of executing the tasks described
```

Comment:
Suggestion for clarity: "Develop an HTTP-based API capable of executing the GET and POST tasks described..."